loopchain/channel/channel_service.py

```diff
@@ -643,9 +634,3 @@ def start_shutdown_timer_when_fail_subscribe(self):
 
     def stop_shutdown_timer_when_fail_subscribe(self):
         self.__timer_service.stop_timer(TimerService.TIMER_KEY_SHUTDOWN_WHEN_FAIL_SUBSCRIBE)
-
-    def start_switch_role_timer_for_last_leader_when_term_expired(self):
-        self.__timer_service.add_timer_convenient(timer_key=TimerService.TIMER_KEY_SHUTDOWN_WHEN_TERM_EXPIRED,
-                                                  duration=conf.SWITCH_ROLE_DELAY_FOR_LAST_LEADER,
-                                                  callback=self.switch_role,
-                                                  callback_kwargs={"force": True})
```

Comment:
`force` parameter in `switch_role(force=True)` using only here.
It deleted, so remove `force` parameter in `switch_role` function.

this comment is just for history.